src/routers/device.py

```diff
@@ -43,6 +43,12 @@ async def read_device_data_alarm(device_id: str = Path(regex=RE_UUID), db: Async
     return result
 
 
+@router.get("/device/{device_id}/notifications", response_model=List[device_schema.DeviceNotificationData])
+async def read_device_data_notifications(device_id: str = Path(regex=RE_UUID), db: AsyncSession = Depends(get_db)):
+    result = await device_crud.get_device_data_notifications(db, device_id)
+    return result
```

Comment:
Delete `device_id: str = Path(regex=RE_UUID),`.
Add `current_user: auth_schema.SystemUser = Depends(get_current_user),` to the input parameter.

Reply:
Pass the `current_user.user_id` to `device_crud.get_device_data_notifications(db, current_user.user_id)`

Reply:
Done. Which imports do I need to add for the above?

Reply:
You can reference from src/routers/settings.py

Reply:
Done, please check!